<launch>
    <include file="$(find zed_wrapper)/launch/zedm.launch" />

    <node pkg="aaw_ros" type="aaw_visualServo" name="aaw_visualServo_doubleView" output="screen">
        <remap from="/image_raw/left" to="/zedm/zed_node/left/image_rect_gray"/>
        <remap from="/image_raw/right" to="/zedm/zed_node/right/image_rect_gray"/>
    </node>
</launch>